modules/planning/tasks/deciders/open_space_decider/open_space_roi_decider.cc

```diff
@@ -449,17 +449,132 @@ void OpenSpaceRoiDecider::GetRoadBoundary(
   }
 }
 
+void OpenSpaceRoiDecider::GetRoadBoundaryFromMap(
+    const hdmap::Path &nearby_path, const double center_line_s,
+    const Vec2d &origin_point, const double origin_heading,
+    std::vector<Vec2d> *left_lane_boundary,
+    std::vector<Vec2d> *right_lane_boundary,
+    std::vector<Vec2d> *center_lane_boundary_left,
+    std::vector<Vec2d> *center_lane_boundary_right,
+    std::vector<double> *center_lane_s_left,
+    std::vector<double> *center_lane_s_right,
+    std::vector<double> *left_lane_road_width,
+    std::vector<double> *right_lane_road_width) {
+  // Longitudinal range can be asymmetric.
+  const double kLongitudinalStartRange = 10.0;
+  const double kLongitudinalEndRange = 20.0;
+  double start_s = center_line_s - kLongitudinalStartRange;
+  double end_s = center_line_s + kLongitudinalEndRange;
+  hdmap::MapPathPoint start_point = nearby_path.GetSmoothPoint(start_s);
+
+  double index = 0.0;
```

Comment:
`index` usually indicated int types, try re-name it to be more precious

Reply:
fixed. thank you.